README.md

```diff
@@ -122,7 +124,19 @@ beforeAll(() => {
 });
 ```
 
-That's it! You're good to go.
+### With Karma and TypeScript in an Angular project
+
+Add to the existing `types` array in `tsconfig.spec.json`: `"@testing-library/jasmine-dom"`
```

Comment:
Add `"@testing-library/jasmine-dom"` to `types` in the tests `tsconfig` (`tsconfig.spec.json` in an Angular project).

Example:

```
compilerOptions:
  types: ["jasmine", "node", "@testing-library/jasmine-dom"],.
```